Dockerfile

```diff
@@ -3,7 +3,7 @@
 #
 # The produced image is published to https://hub.docker.com/r/anzbank/sysl
 
-FROM golang:1.13-alpine
```

Comment:
The reason to use `golang:1.13-alpine` is `go mod` commands which sysl modules is based on require installed `go`.

Reply:
Oh wait, do some of the sysl codegen commands use go mod internally?

Reply:
Yes, if it imports sysl files from external repo.

Reply:
From my understanding, the build process occurs in a different environment as part of the goreleaser workflow, this dockerfile just copies over the binary without running a build?

So, that means that we actually don't need the `golang:1.13-alpine` image, we can just use alpine

Happy to talk about this in person, if I've got the wrong understanding of the dependencies here!

Reply:
> From my understanding, the build process occurs in a different environment as part of the goreleaser workflow, this dockerfile just copies over the binary without running a build?

Yes you are right about this. 
As for the base image, if we use `alpine`, the sysl binary will raise "command not found: go" error when it runs this function https://github.com/anz-bank/sysl/blob/master/pkg/mod/gomod.go#L80. Hence `golang:1.13-alpine` is required.

Reply:
Okay turns out Go Get is used by the Sysl Modules functionality so we should stick with `golang:1.13-alpine`, will remove the changes!